scaleway/resource_rdb_instance_test.go

```diff
@@ -198,6 +198,41 @@ func TestAccScalewayRdbInstance_Settings(t *testing.T) {
 	})
 }
 
+func TestAccScalewayRdbInstance_InitSettings(t *testing.T) {
+	tt := NewTestTools(t)
+	defer tt.Cleanup()
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: tt.ProviderFactories,
+		CheckDestroy:      testAccCheckScalewayRdbInstanceDestroy(tt),
+		Steps: []resource.TestStep{
+			{
+				Config: `
+					resource scaleway_rdb_instance main {
+						name = "test-rdb"
```

Comment:
Can this create a conflict with other tests ?

Reply:
Yes, I'm gonna rename it